apps/src/lib/levelbuilder/rubrics/RubricsContainer.jsx

```diff
@@ -204,10 +226,15 @@ RubricsContainer.propTypes = {
   unitName: PropTypes.string,
   lessonNumber: PropTypes.number,
   levels: PropTypes.arrayOf(
```

Comment:
Thoughts on combining this and the `hasSubmittableLevels` prop? ie pre-filtering `levels` to only contain the submittable levels then deducing `hasSubmittableLevels` from the length of that list?

Reply:
I LOVE this!  I just refactored it and re-committed.